### PR TITLE
Add tags on the page model

### PR DESCRIPTION
**Pull request by zatan**
This branch should merge after `docs` branch successfully merged into the `master` branch.

### CODE REVIEW

**Comment by zatan** [comment]
After merge I need to add this to **lumos** requirements.

File: glitter/pages/admin.py [hunk 1]
```diff
@@ -57,6 +57,10 @@ def get_fields(self, request, obj=None):
         if not getattr(settings, 'GLITTER_SHOW_LOGIN_REQUIRED', False):
             fields.remove('login_required')
 
+        # Show glitter tags if it's set to show.
+        if getattr(settings, 'GLITTER_PAGES_TAGS', False):
```
Comment (tomkins) [inline glitter/pages/admin.py]: Looking at the lines above, shall we keep the adding/removing consistent?

I'd possibly suggest removing, so we can position fields as we want in `fields`, and just remove the bits we don't want.

Comment (zatan) [inline glitter/pages/admin.py]: +1

**Comment by zatan** [comment]
I fixed some issue with the line length and on the next release I would like to see some docs and tags on pages as the feature?

**Comment by tomkins** [comment]
:shipit:

**Pull request merged**